.circleci/config.yml

```diff
@@ -55,10 +55,9 @@ jobs:
       - checkout
 
       - restore_cache:
-          key: v1-py2-{{ checksum "setup.py" }}
-
-      - restore_cache:
-          key: v1-py2-{{ .Revision }}
+          keys:
+            - v1-py2-deps-{{ checksum "setup.py" }}
+            - v1-py2-build-{{ .Revision }}
```

Comment:
Apparently, that's not how Circle works: when you provide two keys, it will checkout the second one _only_ if there is no match for the first one.  We need two separate calls.